app/app.py

```diff
@@ -42,14 +42,12 @@ def migrations():
         return f"Last migration on {last_migration_date}"
 
 def get_db_token(host, port, user):
-    region = os.environ.get("AWS_REGION")
-
     # gets the credentials from .aws/credentials
-    logger.info("Getting RDS client for region %s", region)
-    client = boto3.client("rds", region_name=region)
+    logger.info("Getting RDS client")
+    client = boto3.client("rds")
 
     logger.info("Generating auth token for user %s", user)
-    token = client.generate_db_auth_token(DBHostname=host, Port=port, DBUsername=user, Region=region)
+    token = client.generate_db_auth_token(DBHostname=host, Port=port, DBUsername=user)
```

Comment:
Based on the boto docs, this line will use the client's region if not passed https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/rds/client/generate_db_auth_token.html

As for the client region, I previously thought that setting `AWS_REGION` would cause the boto client to use the correct region when creating the client with `boto3.client(...)`, but that was wrong so in some places we ended up adding `boto3.client(..., region_name=...)`. I later realized when looking at [the boto docs](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#using-environment-variables) that we need to set `AWS_DEFAULT_REGION`, not `AWS_REGION`. So I believe (but to be honest didn't test) that we don't actually need to pass region_name to the boto3.client call as long as AWS_DEFAULT_REGION is set.